elements/lisk-api-client/package.json

```diff
@@ -39,15 +39,15 @@
 		"@liskhq/lisk-cryptography": "^4.0.0-alpha.4",
 		"@liskhq/lisk-transactions": "^6.0.0-alpha.5",
 		"@liskhq/lisk-validator": "^0.7.0-alpha.4",
-		"isomorphic-ws": "4.0.1",
-		"ws": "7.4.6",
+		"isomorphic-ws": "5.0.0",
+		"ws": "8.11.0",
 		"zeromq": "6.0.0-beta.6"
```

Comment:
maybe we can also upgrade zeromq https://github.com/zeromq/zeromq.js/releases/tag/v6.0.0-beta.14

Reply:
I tried to update zeromq as well. However, the newer version fails on building zeromq... 😓